city/models.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import Column, Integer, String
+
+from database import Base
```

Comment:
The `City` class should be defined in this file as it represents the city model for your application. Ensure that the class is defined with the necessary fields as per the task requirements.